doc/Guides/DevelopmentServer.md

```diff
@@ -0,0 +1,239 @@
+
+Click on **Next: Review and Launch"**
+
+* Review
+
+Almost there! Make sure everything is correct and you are using the free tier, then click on **"Launch"**.
+![alt text](https://i.imgur.com/jcwooNO.png)
+
+1 more thing! A window will pop up, asking for your key pair. Just create a new one by selecting the dropdown: **"Create a new key pair"** and
+give it a name, click on **"Download Key Pair"**.
+Keep the key file (.PEM File) somewhere secure (To be 100% certain, back it up somewhere). **YOU CAN NOT ACCESS THE CREATED INSTANCE WITHOUT THIS KEY**
+
+![alt text](https://i.imgur.com/ZP6eJq5.png)
+
+Now you can (finally) click on **Launch Instances**!
+
+Go back to your EC2 dashboard by clicking on "Services" at the top left and clicking on EC2.
+Now you see you have "Running Instances: 1". **Click on "Running Instances" to continue**.
+
+![alt text](https://i.imgur.com/qz4YwgB.png)
+
+You now see your instance running! If it's still initializing, give it a few minutes! A new instance might take around 5-10min to set up.
+Refresh the page after 10 minutes if nothing changes.
+
+Now you did all this but you want to get ON the dedicated server right? Perfect! The next step will get you up and running!
+
+**3. Configuring the RDP(Remote Desktop Program)**
+
+Time to get the RDP file so you can start connecting!
+There are a few things we'll need!
+
+1. RDP file with the key pair added to it
+2. Configure RDP file once downloaded to allow getting files from our C: drive or other drives (so you can easily get your zip project.
+3. Enter the windows Admin password once you start the RDP file
+
+Once you've doen this, you should be able to continue using the same RDP file.
+
+* 1. RDP file with the key pair added to it
+![alt text](https://i.imgur.com/b7YGhX4.png)
+
+**BEFORE CLICKING DOWNLOAD GET THE PASSWORD**
+
+![alt text](https://i.imgur.com/yTqKryT.png)
+
+**COPY THE PASSWORD FOR LATER**
+
+Now click on **"Download Remote Desktop File"**
+
+The RDP file will be downloaded.
+
+![alt text](https://i.imgur.com/SJ0ER7y.png)
+
+* 2. Configure RDP file for easy file access
+
+Go to your freshly downloaded RDP file and rightclick it and then click "Edit".
+
+Go to the third tab "Local sources", at the bottom click "more" under local devices and sources. 
+On the new window select your C: drive or any other drive. This is your own computer your connecting from. For easy file exchange.
+
+![alt text](https://i.imgur.com/2U4MLkS.png)
+
+Perfect! Now you can run the RDP file! The RDP file will ask a password. If you forgot your password you can get it back by rightclicking 
+the instance and clicking on "Get Windows Password". You will be asked to re-enter your key pair (.PEM) file and decrypt the message.
+Once done you will be able to copy the password.
+
+![alt text](https://i.imgur.com/evD4c6s.png)
+
+![alt text](https://i.imgur.com/cYjS94u.png)
+
+There you have it! Now you 100% have your password and you should be logging into your dedicated server!
```

Comment:
Suggestion: Change to...
_There you have it! Now you have your password and Remote Desktop should show you logging into your dedicated server!_

This just sounds a little weird with the 100%, might just be needing another pass.